app/styles/pages/events.scss

```diff
@@ -6,6 +6,11 @@
   margin-top: 10px;
 }
 
+.vertical-item {
```

Comment:
Move it to a separate partial call `tabbed-navigation` and scope it within a class called `tabbed-navigation` and add that class to the component.